Methods.h

```diff
@@ -0,0 +1,50 @@
+#include "Classes.h"
+using namespace std;
+
+void Smartphone::add(float p, string n, float M, float A, float B)
+{
+    name = n;
+    price = p;
+    Memory_capacity = M;
+    Array_of_mobile_numbers = A;
+    Battery_capacity = B;
+}
+void Smartphone::out()
+{
```

Comment:
You are using unclear names for the class methods.

You should:
void Smartphone::out_InformationAboutSmartphone() {